packages/desktop/views/login/views/SelectProfileView.svelte

```diff
@@ -24,7 +24,7 @@
         const isDeveloperProfile = shouldBeDeveloperProfile()
         await initialiseOnboardingFlow({
             isDeveloperProfile,
-            ...(!isDeveloperProfile && { networkId: NetworkId.Shimmer }),
+            ...(!isDeveloperProfile && { networkId: NetworkId.Testnet }),
```

Comment:
Wait, sorry I was wrong about this - it should be `NetworkId.Shimmer` if `isDeveloperProfile` is false?